test/index.js

```diff
@@ -1,9 +1,8 @@
-
 const cp = require('child_process');
 require('should');
 
 test('validate generated queries', async () => {
-  cp.execSync('node index.js --schemaFilePath ./example/sampleTypeDef.graphql --destDirPath ./example/output');
-  const queries = require('../example/output');
-  queries.mutations.signin.indexOf('signin').should.not.equal(-1);
+  cp.execSync('node index.js --schemaFilePath ./example/sampleTypeDef.graphql --destDirPath ./example/output --depthLimit 3');
+  const query = require('../example/output');
+  query.mutation.signin.indexOf('signin').should.not.equal(-1);
```

Comment:
If not must, it will be better not changing the current API, as people are already using it